src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = true;
+  }
 }
 
+Animal.alive = [];
+
 class Carnivore extends Animal {
-  // write your code here
+  bite(herbivoreAnimal) {
+    const eatenAnimal = herbivoreAnimal;
```

Comment:
Are U sure U need here an extra variable?)

Reply:
Agree, it's redundant
```suggestion
```